### PR TITLE
[TFA]: replacing topic list with topic get for testing put empty bucket notifications

### PR DESCRIPTION
**Pull request by hmaheswa**
replacing topic list with topic get for testing put empty bucket notifications

automation failure log:
http://magna002.ceph.redhat.com/cephci-jenkins/test-runs/18.2.0-128/Weekly/rgw/16/tier-2_rgw_test_bucket_notifications/put_empty_bucket_notifications_0.log

as mentioned in this comment:
https://bugzilla.redhat.com/show_bug.cgi?id=2239173#c8
"radosgw-admin topic list --bucket bucket_name" command was listing autogenerated notification topic only associated to that bucket. (which was a bz)
with the fix in the above bz, listing autogenerated notification topics in "radosgw-admin topic list" is removed.
so modifying the topic list command to topic get through this PR.

pass logs: http://magna002.ceph.redhat.com/ceph-qe-logs/Hemanth_Sai/empty_notif_topic_list_PR_logs/cephci-run-V9T7GL/

### CODE REVIEW

**Review by anrao19 — APPROVED** [review]

**Review by ckulal — COMMENTED** [review]
LGTM

File: rgw/v2/tests/s3_swift/test_bucket_notifications.py [hunk 1]
```diff
@@ -350,16 +349,19 @@ def test_exec(config, ssh_con):
...
+                    get_rgw_notif_topic = notification.rgw_topic_ops(
+                        op="get",
+                        args={"topic": bkt_notif_topic_name, **extra_topic_args},
                     )
-                    log.info(topics_list["topics"])
-                    if topics_list["topics"]:
+                    if get_rgw_notif_topic:
                         raise TestExecError(
-                            "radosgw-admin topic list is not empty even after put empty notifications"
+                            "radosgw-admin topic get for autogenerated notification topic is successful even after put empty notifications"
```
Comment (mkasturi18) [inline rgw/v2/tests/s3_swift/test_bucket_notifications.py:360]: @hmaheswa , can we reword it to reflect the exact failure
